server/src/routes/api/v1/trailsRouter.js

```diff
@@ -20,6 +20,7 @@ trailsRouter.get("/:id", async (req, res) => {
   const { id } = req.params
   try {
     const trail = await Trail.query().findById(id)
+    trail.reviews = await trail.$relatedQuery("reviews")
```

Comment:
We should have a trail serializer and a review serializer! Within the Trail serializer is where we can query for the associated reviews - passing them through the Review serializer

Reply:
Awesome! We've got some tasty nested serializers ready on the grill for ya